source/_integrations/netatmo.markdown

```diff
@@ -87,32 +87,81 @@ The `netatmo` switch platform provides support for Legrand/BTicino switches and
 
 `set_camera_light_mode`
 
-Set the outdoor camera light mode. This requires an entity id and a valid state.
+Set the outdoor camera light mode.
+
+| Service data attribute | Required | Description                |
+| ---------------------- | -------- | -------------------------- |
+| `camera_light_mode`    | Yes      | Outdoor camera light mode. |
 
 ### Set Schedule
 
 `set_schedule`
 
-Set the heating schedule. This requires an entity id and a schedule name.
+Set the heating schedule.
+
+| Service data attribute | Required | Description                           |
+| ---------------------- | -------- | ------------------------------------- |
+| `schedule_name`        | Yes      | The name of the schedule to activate. |
 
-### Set preset mode with end datetime
+### Set preset mode with end date & time
 
 `set_preset_mode_with_end_datetime`
 
 Set the preset mode for a Netatmo climate device. The preset mode must match a preset mode configured at Netatmo.
 
+| Service data attribute | Required | Description                                                |
+| ---------------------- | -------- | ---------------------------------------------------------- |
+| `preset_mode`          | Yes      | Climate preset mode such as Schedule, Away or Frost Guard. |
+| `end_datetime`         | Yes      | Date & time the preset will be active until.               |
+
+### Set temperature with end date & time
+
+`set_temperature_with_end_datetime`
+
+Sets the target temperature for a Netatmo climate device with an end date & time.
+
+| Service data attribute | Required | Description                                              |
+| ---------------------- | -------- | -------------------------------------------------------- |
+| `target_temperature`   | Yes      | The target temperature for the device.                   |
+| `end_datetime`         | Yes      | Date & time the target temperature will be active until. |
+
+### Set temperature with time period
+
+`set_temperature_with_time_period`
+
+Sets the target temperature for a Netatmo climate device with a time period it will be active for.
+
+| Service data attribute | Required | Description                                            |
+| ---------------------- | -------- | ------------------------------------------------------ |
+| `target_temperature`   | Yes      | The target temperature for the device.                 |
+| `time_period`          | Yes      | Time period the target temperature will be active for. |
+
+### Clear Temperature Setting
```

Comment:
```suggestion
### Clear temperature setting
```
We use sentence-style capitalization. I know there is heading style capitalization elsewhere in this text. But I think those were fixed on the current branch.